joshprzybyszewski/cribbage
# Add sidebar

## What broke / What you're adding
Resolves #74. It does not appear on the landing page, but appears anytime someone is logged in. Here's what it looks like:
![image](https://user-images.githubusercontent.com/19275178/87259319-25561080-c470-11ea-989c-7aa2e7baf782.png)



The color palette probably isn't optimal, but it's a start. At the moment, the links take you to a few empty pages that will get filled in later.

## How you did it
UPDATE: we're using [material UI](https://material-ui.com/components/material-icons/) from google now. 

## How to test it and how to try to break it
Checkout this branch and run the server/client. Log into the app to see the sidebar. Try clicking each icon to make sure they take you places.
